src/main/java/net/syscon/elite/repository/impl/OffenderDeletionRepositoryImpl.java

```diff
@@ -406,24 +406,20 @@ private void deleteOffenderBeneficiaries(final Set<String> offenderIds) {
     }
 
     private int executeNamedSqlWithOffenderIds(final String sql, final Set<String> ids) {
-        log.debug("Executing deletion SQL: {}, with ids: {}", sql, ids);
         return jdbcTemplate.update(getQuery(sql), createParams("offenderIds", ids));
     }
 
     private void executeNamedSqlWithBookingIds(final String sql, final Set<String> ids) {
-        log.debug("Executing deletion SQL: {}, with ids: {}", sql, ids);
         jdbcTemplate.update(getQuery(sql), createParams("bookIds", ids));
     }
 
-    private int executeNamedSqlWithOffenderIdsAndBookingIds(final String sql,
-                                                            final Set<String> offenderIds,
-                                                            final Set<String> bookIds) {
-        log.debug("Executing deletion SQL: {}, with offender ids: {} and bookIds: {}", sql, offenderIds, bookIds);
-        return jdbcTemplate.update(getQuery(sql), createParams("offenderIds", offenderIds, "bookIds", bookIds.isEmpty() ? null : bookIds));
+    private void executeNamedSqlWithOffenderIdsAndBookingIds(final String sql,
+                                                             final Set<String> offenderIds,
+                                                             final Set<String> bookIds) {
+        jdbcTemplate.update(getQuery(sql), createParams("offenderIds", offenderIds, "bookIds", bookIds.isEmpty() ? null : bookIds));
```

Comment:
More out of curiosity.  Under what circumstances would the booking id's be empty?

Reply:
Sometimes an offender record can be created without an associated booking (if a prison officer starts to create one before realising that there's already an exisiting record for example). So this allows for that.